plugins/examples/security-authorization-engine/src/test/java/org/elasticsearch/example/CustomAuthorizationEngineTests.java

```diff
@@ -118,7 +118,7 @@ public void testAuthorizeIndexAction() {
         CustomAuthorizationEngine engine = new CustomAuthorizationEngine();
         Map<String, IndexAbstraction> indicesMap = new HashMap<>();
         indicesMap.put("index", new ConcreteIndex(IndexMetadata.builder("index")
-            .settings(Settings.builder().put("index.version.created", Version.CURRENT))
+            .settings(Settings.builder().put("index.version.created", IndexVersion.current()))
```

Comment:
Can we change this to `IndexMetadata.SETTING_VERSION_CREATED` too?

Reply:
This is in a plugin example, not compiled code, so I don't want to introduce class references that might not be there when this is used